bazel/envoy_build_system.bzl

```diff
@@ -20,12 +20,45 @@ def envoy_copts(repository, test = False):
         repository + "//bazel:dbg_build": ["-ggdb3"],
     }) + select({
         repository + "//bazel:disable_tcmalloc": [],
+        "//bazel/osx:darwin": [],
         "//conditions:default": ["-DTCMALLOC"],
     }) + select({
         repository + "//bazel:disable_signal_trace": [],
         "//conditions:default": ["-DENVOY_HANDLE_SIGNALS"],
     })
 
+# Compute the final linkopts based on various options.
+def envoy_linkopts():
+    return select({
+        "//bazel/osx:darwin": [],
```

Comment:
Personally I would just put in a TODO. This is really only useful in a production setting when dealing with core dumps.